microsoft/react-native-macos
# Adds AccessibilityInfo.isScreenReaderEnabled to RN MacOS

#### Please select one of the following

## Summary

When the MacOS fork was made all the AccessibilityInfo functionality was stubbed out because the iOS Accessibility APIs are different from the MacOS APIs. I've filled back out the functionality for isScreenReaderEnabled, and the corresponding event screenReaderChanged.

Previously the Native side implementation of the AccessibilityManager was surrounded by a big ifdef to make sure it was only actually implemented for the supported iOS platform. I've now forked the file so it has a MacOS version. This currently only exports the one method and the one event-listener, but that will change as more features are requested.

## Changelog

[MacOS] [Added] - Implemented AccessibilityInfo.isScreenReaderEnabled and screenReaderChanged

## Test Plan

1. Run RNTester
2. Go to the accessibility page 
3. Scroll to screen reader enabled (second to last)
4. Toggle VoiceOver in settings.
- You should see the text change back and forth between "The screen reader is disabled" and "The screen reader is enabled"

![Screen Shot 2020-06-10 at 12 27 29](https://user-images.githubusercontent.com/65255457/84309989-cbcb9080-ab15-11ea-97b4-50c2cc92c69a.png)
![Screen Shot 2020-06-10 at 12 27 37](https://user-images.githubusercontent.com/65255457/84309992-ce2dea80-ab15-11ea-9db7-e61a7e901d23.png)


###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-macos/pull/444)
